domaci22.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+<link rel="stylesheet" href="stil22.css"> 
```

Comment:
Link files with a `./` prefix. Example: `./stil22.css`. On a real server, this might not work.

Reply:
Also, name your files in a way other developer would know what's happening. In this case, if this is second version of css, name it `stil-version-two`. Actually, you can write it in the same file without renaming, git tracks the changes for us :)